docs/8. FAQ/Resolving 400 Error When Resuming a Flow via API.md

```
@@ -0,0 +1,63 @@
+## Why do I get a 400 error when resuming a flow via API?
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)